nano.save

```diff
@@ -1 +0,0 @@
-Node send email to user on the basis of tag id with preconfigured template.
```

Comment:
i dont understand. remove this file. or put in gitignore. this should not come in pr